frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -148,24 +156,35 @@ const StoryCard = ({
         allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
         allowFullScreen
       />
-      <div className="story-card-details">
-        <div className="story-card-top">
-          <p className="story-card-title">{title}</p>
-          <div className="story-card-tags">
-            <Tag displayText={`Level ${level}`} />
-            <Tag displayText={`${convertLanguageTitleCase(language)}`} />
-          </div>
-        </div>
-        <p className="story-card-description">{description}</p>
-      </div>
-      <div className="story-card-actions">
-        <PrimaryButton
+      <Box flexBasis="40%" padding="17px 0px 17px 26px">
+        <Flex direction="column" wrap="wrap">
+          <Heading size="md">{title}</Heading>
+          <Flex direction="row">
+            <Badge>{`Level ${level}`}</Badge>
+            <Badge>{`${convertLanguageTitleCase(language)}`}</Badge>
+          </Flex>
+        </Flex>
+        <Text>{description}</Text>
+      </Box>
+      <Box flexBasis="40%" paddingTop="30px" textAlign="center">
+        <Button
+          colorScheme="blue"
+          variant="solid"
+          size="wide"
           onClick={primaryBtnOnClick()}
-          displayText={primaryBtnText()}
-        />
-        <SecondaryButton onClick={previewBook} displayText="preview book" />
-      </div>
-    </div>
+        >
+          {primaryBtnText()}
+        </Button>
+        <Button
+          colorScheme="blue"
+          variant="outline"
+          size="wide"
```

Comment:
If these traits will be common to a lot of buttons, we should be able to include this in the `customTheme`